dali/base/dasess.cpp

```diff
@@ -1405,7 +1462,15 @@ class CCovenSessionManager: public CSessionManagerBase, implements ISessionManag
         return NULL;
     }
 
+    //ISessionManager
     virtual SecAccessFlags getPermissionsLDAP(const char *key,const char *obj,IUserDescriptor *udesc,unsigned flags, int *err)
+    {
+        UNIMPLEMENTED;//client version of getPermissionsLDAP from ISessionManager
+        return SecAccess_Unavailable;
+    }
```

Comment:
See note where SMSgetPermissionsLDAP is currently called.
I think it should call common getPermissionsLDAP and that method should default the new params to null.
That way, this doesn't have to be unimplemented, i.e. a server-side could still be called (without signature).

Reply:
@RussWhitehead - I think this one missed, can be removed, if new params are defaulted to null, thus allowing any server caller to call getPermissionsLDAP without a signature.